### PR TITLE
PUB-543 - Admin Dashboard

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-543

### Change description ###
- create admin dashboard with 4 items 50% width
- located on the `admin-dashboard` protected route
- route items to appropriate pages (PUB-544 - manual upload , file remove - #268 , PUB-1063 to be developed, PUB-1055 to be developed)

**NOTE:**
E2e test only added for manual upload, the rest to be added after ticket is merged in / developed
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
admindash

### CODE REVIEW

**Review by alberto-hortiguela — APPROVED** [review]

File: src/test/e2e/PageObjects/AdminDashboard.page.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { CommonPage } from './Common.page';
+import { ManualUploadPage } from './ManualUpload.page';
+
+const helpers = require('../Helpers/Selectors');
+
+export class AdminDashboardPage extends CommonPage {
+  async clickUploadFileCard(): Promise<ManualUploadPage> {
+    await $(helpers.UploadFile).catch(() => {
+      console.log(`${helpers.EmailSubscriptionLink} not found`);
```
Comment (ChrisS1512) [inline src/test/e2e/PageObjects/AdminDashboard.page.ts:9]: Think this needs updating from EmailSubscriptionLink

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by kurt-uk — APPROVED** [review]

File: src/test/e2e/PageObjects/AdminDashboard.page.ts (hunk 1 above)
Comment (kurt-uk) [reply src/test/e2e/PageObjects/AdminDashboard.page.ts:9]: ^^

**Review by furnivall — APPROVED** [review]
Looks good!

File: src/test/e2e/PageObjects/AdminDashboard.page.ts (hunk 1 above)
Comment (commitBlob) [reply src/test/e2e/PageObjects/AdminDashboard.page.ts]: Whoops, should be fixed now

**Review by marcogagliardi — DISMISSED** [review]
👌🏽

**Review by furnivall — APPROVED** [review]
looks good :)

**Pull request merged**